src/Mod/Draft/draftutils/todo.py

```diff
@@ -110,12 +110,12 @@ def doTasks():
 
         The lists are `itinerary`, `commitlist` and `afteritinerary`.
         """
-        print("Debug: doing delayed tasks.\n"
-              "itinerary: {0}\n"
-              "commitlist: {1}\n"
-              "afteritinerary: {2}\n".format(todo.itinerary,
-                                             todo.commitlist,
-                                             todo.afteritinerary))
+#        print("Debug: doing delayed tasks.\n"
+#              "itinerary: {0}\n"
+#              "commitlist: {1}\n"
+#              "afteritinerary: {2}\n".format(todo.itinerary,
+#                                             todo.commitlist,
+#                                             todo.afteritinerary))
```

Comment:
Maybe you can do it like this:
```py
        DEBUG = 0
        if DEBUG:
            print("Debug: doing delayed tasks.\n"
                  "itinerary: {0}\n"
                  "commitlist: {1}\n"
                  "afteritinerary: {2}\n".format(todo.itinerary,
                                                 todo.commitlist,
                                                 todo.afteritinerary))
```
Then somebody can change the value to `DEBUG = 1` to reactivate the debug information quickly.

Reply:
good idea...